coriolisclient/tests/cli/test_endpoint_destination_minion_pool_options.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2024 Cloudbase Solutions Srl
+# All Rights Reserved.
+
+from cliff import lister
+from unittest import mock
+
+from coriolisclient.cli import endpoint_destination_minion_pool_options
+from coriolisclient.cli import utils as cli_utils
+from coriolisclient.tests import test_base
+
+
+class EndpointDestinationMinionPoolOptionsFormatterTestCase(
+        test_base.CoriolisBaseTestCase):
+    """
+    Test suite for the Coriolis Client Endpoint Destination Minion Pool
+    Options Formatter.
+    """
+
+    def setUp(self):
+        super(EndpointDestinationMinionPoolOptionsFormatterTestCase, self
+              ).setUp()
+        self.endpoint = (endpoint_destination_minion_pool_options.
+                         EndpointDestinationMinionPoolOptionsFormatter)()
+
+    def test_get_sorted_list(self):
+        obj1 = mock.Mock()
+        obj2 = mock.Mock()
+        obj3 = mock.Mock()
+        obj1.name = "app1"
+        obj2.name = "app2"
+        obj3.name = "app3"
+        obj_list = [obj2, obj1, obj3]
+
+        result = self.endpoint._get_sorted_list(obj_list)
+
+        self.assertEqual(
+            [obj1, obj2, obj3],
+            result
+        )
+
+    @mock.patch.object(cli_utils, 'format_json_for_object_property')
+    def test_get_formatted_data(self, mock_format_json_for_object_property):
+        obj = mock.Mock()
+        obj.name = mock.sentinel.name
+        obj.to_dict = mock.Mock(
+            return_value={"config_default": mock.sentinel.config_default}
+        )
+
+        result = self.endpoint._get_formatted_data(obj)
+
+        self.assertEqual(
+            (
+                mock.sentinel.name,
+                mock_format_json_for_object_property.return_value,
+                mock.sentinel.config_default
+            ),
+            result
+        )
+
+
+class ListEndpointDestinationMinionPoolOptionsTestCase(
+        test_base.CoriolisBaseTestCase):
+    """
+    Test suite for the Coriolis Client List Endpoint Destination Minion Pool
+    Options.
+    """
+
+    @mock.patch.object(lister.Lister, '__init__')
+    def setUp(self, mock__init__):
+        mock__init__.return_value = None
+        super(ListEndpointDestinationMinionPoolOptionsTestCase, self).setUp()
+        self.endpoint = (endpoint_destination_minion_pool_options.
+                         ListEndpointDestinationMinionPoolOptions)()
```

Comment:
This class's inherited `__init__` takes two positional arguments: `app` and `app_args`. Please pass those too here, for safety reasons:
```suggestion
        self.mock_app = mock.Mock()
        self.endpoint = (
            endpoint_destination_options.ListEndpointDestinationOptions(
                self.mock_app, mock.sentinel.app_args))
```